perf/src/main/java/org/bf2/performance/InstanceProfiler.java

```diff
@@ -562,7 +561,7 @@ protected LatencyResult determineLatency(Profile profile, Consumer<Workload> set
             result.medianEndToEndLatency99pct = TestUtils.getMedian(loadTestResult.endToEndLatency99pct);
             result.aggregatedPublishLatency50pct = loadTestResult.aggregatedPublishLatency50pct;
             result.aggregatedPublishLatency99pct = loadTestResult.aggregatedPublishLatency99pct;
-            result.maxConnectionCount = loadTestResult.additionalMetrics.get(KafkaBenchmarkDriverWithMetrics.CONNECTION_COUNT).stream().max(Double::compareTo).get();
+            result.maxConnectionCount = loadTestResult.additionalMetrics.get(KafkaBenchmarkDriverWithMetrics.CONNECTION_COUNT).stream().max(Double::compareTo).orElse(0.0);
```

Comment:
@MikeEdgar Also wanted to double-check this `orElse()` here. Does this seem fine?

Reply:
@biswassri - sorry, forgot to comment here. Let's leave this as-is and add the `//NOSONAR` comment too.

Reply:
Sure!